packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -214,6 +214,12 @@ class ChromiumLauncher {
       '--no-default-browser-check',
       '--disable-default-apps',
       '--disable-translate',
+
+      // Remove the search engine choice screen. It's irrelevant for app
+      // debugging purposes.
+      // See: https://github.com/flutter/flutter/issues/153928
+      '--disable-search-engine-choice-screen',
```

Comment:
Weren't there other places (i.e. where we have `--no-default-browser-check` as well) that could also benefit from this?
(for example the web benchmarks?)

Reply:
Good point. I should have checked more carefully. However, now that I did check, it doesn't seem like it's necessary in those other places. The benchmark code is ad hoc for Flutter's own testing, and we use Chromium for that, where I don't think this is an issue. For `flutter drive` we use WebDriver, and I do not expect driver-initialized browser to put anything that would prevent tests from running. We may have to revisit if it comes up though.